doc/rst/platforms/cray.rst

```diff
@@ -666,6 +607,20 @@ synonymous with ``GASNET_MAX_SEGSIZE``, and the former overrides the
 latter if both are set.
 
 
+-------------------------------------
+Special Notes for HPE Cray EX Systems
+-------------------------------------
+
+The gasnet communication layer has not yet been built or tested on EX
+systems.  We expect to provide it in the pre-built module no later than
+the 1.25.0 release.
```

Comment:
I don't think we need to set a date for gasnet support. I would just leave it as "The gasnet communication layer has not yet been built or tested on EX systems." for now

Reply:
Yeah, this was straw wording.  However, I do feel like we should say whether or not we expect to support it and if we do, something about when although it doesn't need to be a release number.  (And similarly for the PALS one, next.)

Reply:
"The gasnet communication layer has not yet been built or tested on EX systems though we expect to add support in the future"?

Reply:
Fair enough -- I'll do that.

Reply:
"add support and build it into the module"?

Reply:
Nah, it doesn't scan as well as I'd thought.  Sticking with just "add support".